pandemania/src/components/Login/Login.module.css

```diff
@@ -0,0 +1,8 @@
+h1 {
+	text-align: center;
+}
+
+.title{
+	font-size: 1rem;
+	margin: 0 0.5rem 0;
+}
```

Comment:
Design is not so important bu I have a proposition:
![image](https://user-images.githubusercontent.com/24575781/107991385-cc86c180-6fd6-11eb-9b43-c8d999381a6c.png)

Or maybe use a material-ui design. (https://material-ui.com/)